Makefile.toml

```diff
@@ -4,14 +4,15 @@ skip_core_tasks = true
 [env]
 BUILDSYS_ARCH = { script = ["uname -m"] }
 BUILDSYS_ROOT_DIR = "${CARGO_MAKE_WORKING_DIRECTORY}"
-BUILDSYS_OUTPUT_DIR = "${BUILDSYS_ROOT_DIR}/build"
-BUILDSYS_PACKAGES_DIR = "${BUILDSYS_OUTPUT_DIR}/packages"
+BUILDSYS_BUILD_DIR = "${BUILDSYS_ROOT_DIR}/build"
+BUILDSYS_PACKAGES_DIR = "${BUILDSYS_BUILD_DIR}/rpms"
 BUILDSYS_TOOLS_DIR = "${BUILDSYS_ROOT_DIR}/tools"
 BUILDSYS_SOURCES_DIR = "${BUILDSYS_ROOT_DIR}/sources"
 BUILDSYS_TIMESTAMP = { script = ["date +%s"] }
 BUILDSYS_VERSION_BUILD = { script = ["git describe --always --dirty --exclude '*' || echo 00000000"] }
 BUILDSYS_VERSION_IMAGE = { script = ["awk -F '[ =\"]+' '$1 == \"version\" {print $2}' Release.toml"] }
 BUILDSYS_VARIANT = "aws-k8s-1.15"
+BUILDSYS_OUTPUT_DIR = "${BUILDSYS_BUILD_DIR}/images/${BUILDSYS_ARCH}-${BUILDSYS_VARIANT}"
```

Comment:
We could rename this to `BUILDSYS_IMAGES_DIR` to better reflect what type of output we're getting.

Reply:
Technically it's not just images, and I didn't want to start a separate debate about where migrations would live, and whether migrations would only be in an architecture-specific directory instead of arch&variant-specific, etc...  Zac and I talked about this but thought it'd be better to leave as a separate decision and possible separate change, and that backfilled this variable naming...  :|